config/locales/en.yml

```diff
@@ -35,7 +35,7 @@ en:
   feedback: "Feedback"
   issue_contact: "Please contact the SUCCESS Center at (843) 792-8300 or success@musc.edu for assistance with this process or with any questions you may have."
   portal_title: "SPARC Request Portal"
```

Comment:
Please do not make changes to this file, we need to maintain out own configuration. If you look in the readme, we show you how to set your own file up for localization.